php-8.1.yaml

```diff
@@ -330,25 +330,32 @@ test:
         - postgresql-client
         - wolfi-base
         - shadow
-        - sudo-rs
         - glibc-locales
         - php-8.1-pgsql
+    accounts:
+      groups:
+        - groupname: wolfi
+          gid: 65532
+      users:
+        - username: wolfi
+          gid: 65532
+          uid: 65532
+      run-as: 65532
     environment:
       PGDATA: /tmp/test_db
       PGUSER: wolfi
   pipeline:
     - name: "Test database creation"
       runs: |
-        useradd $PGUSER
-        sudo -u $PGUSER initdb -D /tmp/test_db
-        sudo -u $PGUSER pg_ctl -D /tmp/test_db -l /tmp/logfile start
-        sudo -u $PGUSER createdb testdb
+        initdb -D /tmp/test_db
+        pg_ctl -D /tmp/test_db -l /tmp/logfile start
+        createdb testdb
         psql -lqt | cut -d \| -f 1 | grep -qw testdb
     - name: "Test basic read/write operations"
       runs: |
```

Comment:
```suggestion
```

Reply:
If you want to be able to talk to the postgres server, you need to do it in the same pipeline step.